tensorboard/main.py

```diff
@@ -63,6 +64,9 @@ def run_main():
     raise AssertionError("absl.app.run() shouldn't return")
   except ImportError:
     pass
+  except base_plugin.FlagsException as e:
+    print("Error: " + str(e))
+    sys.exit(-1)
```

Comment:
Exiting with `-1` is misleading; the exit status is a u8, so exiting
with `-1` actually yields a WEXITSTATUS of 255. Why not `sys.exit(1)`?

Reply:
Changed. This was an old habit! In fact I didn't know that it was a u8.